examples/waterwave.py

```diff
@@ -0,0 +1,108 @@
+# waterwave.py
+
+from random import randrange, random
+import taichi as ti
+import numpy as np
+
+ti.init(arch=ti.opengl)
+
+bg = 'misc/test_image.png'
+bg = 'misc/test_small.png'
+bg = ti.imread(bg, 3)
+bg = np.sum(bg, axis=2)
+bg = bg.astype(np.float32) / (256 * 3)
+bg = np.flip(bg, axis=0)
+bg = bg.transpose()
+
+light_color = 1
+kappa = 2
+gamma = 0.6
+eta = 1.333
+inv_eta = 1 / eta
+inv_eta2 = inv_eta**2
+height = 2.5
+dx = 0.02
+dt = 0.01
+inv_dx = 1 / dx
+inv_dx2 = inv_dx**2
+shape = bg.shape
+aspect = shape[0] / shape[1]
+n = shape[0]
+inv_aspect = 1 / aspect
+inv_n = 1 / n
+pixels = ti.var(dt=ti.f32, shape=shape)
+background = ti.var(dt=ti.f32, shape=shape)
+position = ti.var(dt=ti.f32, shape=shape)
```

Comment:
Since you are simulating a height field, is `height` a better name than `position`?

Reply:
Could you fix this?